ReportingServicesTools/Functions/Admin/Set-RSService.ps1

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+
+function Set-RSService
+{
+    <#
+        .SYNOPSIS
+            This command configures service account for PBI and Reporting Server.
+
+        .DESCRIPTION
+            This command configures service account for PBI and Reporting Server.
+
+        .PARAMETER ServiceAccount
+            Specify the domain account reporting services will run under.
+            Use the "ServiceAccount Domain\JonhDow" string to set/update a default value.
+
+        .PARAMETER Password
+            Specify the password to use when running Reporting Services Instance.
+            This is an optional string as its not needed when using managed service accounts or gMSA.
+
+        .PARAMETER ReportServerInstance
+            Specify the name of the SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER ReportServerVersion
+            Specify the version of the SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+
+        .EXAMPLE
+            PS C:\> Set-RSService -ServiceAccount Domain\JohnDow -Password -ReportServerInstance SSRS -ReportServerVersion SQLServer2014
+
+            PS C:\> Set-RSService -ServiceAccount Domain\JohnDow -Password "MySuperPassw0rd!" -ReportServerInstance SSRS -ReportServerVersion SQLServer2016
+
+            This will set the User Account and password Reporting Services runs under.
+    #>
+    
+    [CmdletBinding()]
+    param (
+        [Parameter()]
+        [ValidateNotNullOrEmpty()]
+        [string]
+        $ServiceAccount,
+
+        [string]
+        $Password,
+        
+        [Alias('ReportServerCredentials')]
+        [System.Management.Automation.PSCredential]
+        $Credential,
+
+        [Alias('SqlServerInstance')]
```

Comment:
for new functions you don't need the alias, those are for old functions that wants to provide backward compatibility